### PR TITLE
Remove extra words in firststeps.md (fixes #577)

### PR DESCRIPTION
**Pull request by Loshma93**

### CODE REVIEW

File: pages/firststeps.md [hunk 1]
```diff
@@ -18,26 +18,26 @@ Because these steps are simple, we expect high quality work, which may take a lo
...
+1.Send us your resume again (along with a short description of yourself and what you would like to be called) to vi@ole.org.
+2. Watch [Open Learning Exchange](https://github.com/open-learning-exchange/open-learning-exchange.github.io) and Follow [Emily](https://github.com/EmilyLarkin), [Stuart](https://github.com/sente) and [Dogi](https://github.com/dogi) on GitHub. In GitHub we "Watch" organization repositories and "Follow" individuals. When you are on a repository or profile page, look for these buttons in the top right of the screen.  Also follow each other to see what others in the group are doing.
+3. We use Crowdin to translate our user interface in many different languages. If you know another language, Go to [this Crowdin link](https://crowdin.com/project/open-learning-exchange/invite) to join Crowdin by creating an account. It would be very helpful for you to translate some of the words and familiarize yourself with the software. It would be great to get at least 10% on a language that you speak. If you want to spend additional time, feel free to provide translations to 100%.
```
Comment (waryhermit) [inline pages/firststeps.md]: `Go` does not need to be capitalized since it follows the comma.

File: pages/firststeps.md [hunk 2]
```diff
@@ -49,13 +49,13 @@ Follow the instructions under the [BeLL-Apps tab](bellapps.md).
...
+Create three issues, add comments to three other issues, and resolve three issues by making the necessary changes in your repo with three separate pull requests. You will follow the same steps as in [GitHub Issues](githubissues.md), but will be working to improve this Markdown Wiki so that it is clearer and better for future use. As you have now followed along with the Markdown Wiki thus far, you should all try to improve the clarity of the explanations for these steps. Learn by teaching - or editing our teaching tool, in this case! When you are fixing an issue, Mention the number of the issue you are fixing in the description of your pull request.
```
Comment (waryhermit) [inline pages/firststeps.md]: `Mention` does not need to be capitalized for the same reason.

File: pages/firststeps.md [hunk 3]
```diff
@@ -49,13 +49,13 @@ Follow the instructions under the [BeLL-Apps tab](bellapps.md).
...
 
 ##Step 9 - Submit Survey:
-Let us know in the Gitter chat when you're done, so we can send the survey to your community. Then, fill out the survey and submit it. Next, Sync your community with the nation (as you did in step 7). Lastly, once you have completed all other steps, please add yourself to the virtual intern list found in team.md and create a pull request.
+Let us know in the Gitter chat when you're done, so we can send the survey to your community. Then, fill out the survey and submit it. Next, Sync your community with the nation (as you did in step 7). Lastly, once you have completed all other steps, Add yourself to the virtual intern list found in team.md and create a pull request.
```
Comment (waryhermit) [inline pages/firststeps.md]: `Add` does not need to be capitalized.

**Comment by hannahgdubin** [comment]
I believe there was already a PR approved for this issue? @waryhermit am I correct?

**Comment by waryhermit** [comment]
@hannahgdubin I closed the duplicate, this is for another page it seems. This one is for https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/576

**Comment by Loshma93** [comment]
@waryhermit I made the changes requested

**Pull request merged**